course/frontend/array-shuffle/scripts/main.js

```diff
@@ -0,0 +1,36 @@
+// # Challenge ShuffleArray!
+
+/**
+ * Array.random() Crea un clon exacto de un array y desordena aleatoriamente sus elementos.
+ *
+ * @code
+ *  var originalArray = [1,2,3,'a','b','c'];
+ * 	var newArray = originalArray.random();
+ * @endcode
+ *
+ * @return {array} clonedArray. Retorna un nuevo array.
+ */
+Array.prototype.random = function (){
```

Comment:
RENAME random to shuffle

Reply:
In Trello you said
> LIL polyfill Array.prototype.random() to randomly disorder an array into a new array.

Observe you use the word `random` instead of `shuffle`